translations/wesnoth-A_New_Order/cs.po

```diff
@@ -227,11 +216,12 @@ msgstr ""
 "\n"
 "Poznámky: "
 
+# Nevím, co je za typ terénu farmland, nejspíš neaktualizováno k současné verzi Wesnothu.
```

Comment:
So, this string is currently unused and may be removed (see bugs #55 and #57), so I wouldn't worry about it too much.

Reply:
yeah and actually from looking at my copy of it, it looks like I have actually already removed it... you might want to regenerate the potfile and re-run `msgmerge`